### PR TITLE
Eb/de 42/use data vpc

### PR DESCRIPTION
**Pull request by ethanbrown3**
1. Created variable for team whitelisted ips
2. updated postgres version
3. updated secret storage

### CODE REVIEW

File: aws/alb.tf [hunk 1]
```diff
@@ -27,12 +23,7 @@ resource "aws_security_group" "load_balancer_security_group" {
     to_port     = 443
     protocol    = "tcp"
     description = "dataeng team ips"
-    cidr_blocks = [
-      "50.168.68.90/32", # office
-      "76.140.96.213/32",
-      "67.2.180.194/32",
-      "73.63.25.63/32"
-    ]
+    cidr_blocks = var.whitelist_cidr_blocks
```
Comment (mason-nursa) [inline aws/alb.tf]: Does this include the VPN? Or do I need to send my IP still?

File: aws/mage-prod.tfvars [hunk 2]
```diff
@@ -7,3 +7,9 @@ common_tags = {
   Service   = "mage"
   Repo      = "nursa-com/mage-ai-terraform-templates"
 }
+whitelist_cidr_blocks = [
```
Comment (mason-nursa) [inline aws/mage-prod.tfvars]: Can we put these in a secret or hide them somehow? I don't like the idea of our public ips being stored in the repo

**Review by mason-nursa — CHANGES_REQUESTED** [review]

File: aws/mage-prod.tfvars (hunk 2 above)
Comment (ethanbrown3) [reply aws/mage-prod.tfvars]: Let me look into putting them in secrets

Comment (ethanbrown3) [reply aws/mage-prod.tfvars]: One thing I worry about is when the IPs change we'll need to know what one to remove so we'll need them labeled somehow. I could store them in secrets as a key/value pair then the terraform would have to reference each IP.

Comment (mason-nursa) [reply aws/mage-prod.tfvars]: That could work

File: aws/alb.tf (hunk 1 above)
Comment (ethanbrown3) [reply aws/alb.tf]: No it doesn't. I need to get help from the infra team to figure that out.

**Review by mason-nursa — APPROVED** [review]
Nice

**Pull request merged**